__init__.py

```diff
@@ -102,24 +106,84 @@ def attempt_three_times(function):
                 raise
             msg = "Caught {error}: {message}. Retrying...".format(
                 error=type(e), message=e.message)
-            logger.warning(msg)
+            LOG.warning(msg)
+
+
+#######################################################################
+# Helpers for building the animation frames
+
+def animate(t, often, func, *args):
```

Comment:
After checking out the animation...
- Squiggly lines for the mouth while the speedtest is running is a good idea. 
- The eyes flipping up and down don't add anything to the user experience and make it seem buggy when the eyes are half closed and Mycroft is giving the results at the end. I would remove that and keep them static.